data/human/names.txt

```diff
@@ -692,6 +728,19 @@ phrase "gender-neutral titles"
 		"Admiral"
 		"Sovereign"
 		"Commodore"
+		"President"
+		"Major"
+		"Chieftain"
+		"Overseer"
+		"Cousin"
+		"Steward"
+		"Director"
+		"Prime Minister"
+		"General"
+		"Sergeant"
+		"Hajji"
+		"Rabbi"
+
```

Comment:
Did you mean to add an extra line break here?

Reply:
```suggestion
```